nemo_rl/environments/reward_model_environment.py

```
@@ -0,0 +1,345 @@
+        generation: Generation configuration for VLLM
+    """
+
+    enabled: bool
+    model_name: str
+    precision: str
+    batch_size: int
+    checkpoint_path: str
+    logprob_batch_size: int
+    resources: Dict[str, Any]
+    dtensor_cfg: Optional[Dict[str, Any]] = None
+    dynamic_batching: Optional[Dict[str, Any]] = {"enabled": False}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)